moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -719,151 +729,153 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
               />
             </Box>
           </Grid>
-          <Grid item xs={12}>
```

Comment:
I hate that this is so messy but all of this on down is either linting or adding in the conditional (see lin 732)